src/redux/favoritesRedux.js

```diff
@@ -0,0 +1,32 @@
+/* selectors */
```

Comment:
Do we really need to create a whole new subReducer and state for reducers? :O You already have products in `productsRedux`. You can use it and place the action there.